### ISSUE TITLE
Add retractionwatch data to annotator

### ISSUE THREAD

**Issue by LukasWallrich** [forrtproject/FReD#24]
Retracted citations should also be flagged, using this data: https://api.labs.crossref.org/data/retractionwatch?name@email.org [though this is yet another download that slows the app down, so maybe only when user requests to add that?]

**Comment by LukasRoeseler** [comment]
Also, a simultaneous check for entries in pubpeer, retraction database, reproducibility database, statcheck, and FReD could be useful

**Comment by LukasWallrich** [comment]
Agreed - but I think that should rather live in [papercheck](science verse/papercheck)? Otherwise we start duplicating their work

**Comment by LukasWallrich** [comment]
There is some lack of clarity regarding **reinstated** articles - e.g., https://www.cellphysiolbiochem.com/Articles/000297/ is listed as retracted and reinstated, while most articles that are reinstated are only listed once. I am not sure if that means that we need to remove retracted articles that are reinstated, or if this article was retracted, reinstated and retracted again? The journal does not really help, showing the article as retracted, but with the latest explicit notice stating the opposite (https://www.cellphysiolbiochem.com/Articles/000297/)

This situation is very rare (<20 cases in the current RetractionWatch data), and while we can document this somewhere, I don't think it makes sense to have a notice on outputs for this case that might never occur in FReD) - so, we need to decide whether we rather accept very rare false positives or negatives. I would say it's more important to flag articles, but that is more likely to lead to negative reactions ...

**Comment by LukasWallrich** [comment]
@LukasRoeseler what's your thinking on the two points above? 

Flag expressions of concern? I said no above, but would say yes now.

Worry about reinstated articles that might have been retracted again? I still think no.
